rclpy/rclpy/__init__.py

```diff
@@ -49,3 +74,7 @@ def ok():
 
 def shutdown():
     return _rclpy.rclpy_shutdown()
+
+
+def get_implementation_identifier():
```

Comment:
Should this function have `rmw` in its name?

Below it is referred to as "rclpy implementation" - isn't that actually a "rmw implementation"?